google/field_helpers.go

```diff
@@ -0,0 +1,40 @@
+package google
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const networkLinkTemplate = "projects/%s/global/networks/%s"
+
+var networkLinkRegex = regexp.MustCompile(fmt.Sprintf(networkLinkTemplate, "(.+)", "(.+)"))
```

Comment:
This made me quite uncomfortable; something about printing regex chunks into another regex made me squirm a bit

How do you feel about just inlining the regex here?